entry

```diff
@@ -1,29 +1,83 @@
 #!/bin/sh
-set -e -x
+set -ex
 
 trap exit TERM INT
 
-for src_range in ${SRC_RANGES}; do
+info()
+{
+    echo '[INFO] ' "$@"
+}
+fatal()
+{
+    echo '[ERROR] ' "$@" >&2
+    exit 1
+}
+
+check_iptables_mode() {
+    lsmod | grep "nf_tables" 2> /dev/null
+    if [ $? = 0 ]; then
+        mode=nft
+    else 
+        mode=legacy
+    fi
+}
+
+set_nft() {
+    bin_dir="/sbin"
```

Comment:
Could probably define this up at the top instead of setting it to the same thing in both functions.